robottelo/host_helpers/system_tools.py

```diff
@@ -0,0 +1,92 @@
+import contextlib
+import random
+
+from robottelo.cli.proxy import CapsuleTunnelError
+from robottelo.config import settings
```

Comment:
can you move this import into the function that uses it? That will help to avoid loading the settings too soon.

Reply:
sure I did